composer.json

```diff
@@ -4,7 +4,7 @@
 		"optimize-autoloader": true,
 		"classmap-authoritative": true,
 		"platform": {
-			"php": "8.0"
+			"php": "8.0.2"
```

Comment:
is this correct?

Reply:
It's required otherwise the nextcloud/OCP package can not be installed.
And yes server itself also requires 8.0.2 as it's required by symfony container package:
https://github.com/nextcloud/3rdparty/blob/b3d52b32c65999204aefeb85548f95c76391d632/composer.json#L11